ads/mediaimpact.js

```diff
@@ -25,6 +25,15 @@ import {loadScript} from '../3p/3p';
 
 export function mediaimpact(global, data) {
   global.fif = false;
+  /* eslint google-camelcase/google-camelcase: 0 */
+  global.sas_loadHandler = function(f) {
+    if (!f.hasAd) {
```

Comment:
maybe, for what benefit? Haven't heard about avoid negative check till now and cannot find something about it on web...